Sources/IndiceNetworkClient/Models/APIError.swift

```diff
@@ -9,39 +9,37 @@ import Foundation
 
 public struct APIError: LocalizedError {
     
-    public struct SimpleError: NetworkError {
-        public let status: Int?
-        public let description: String
-    }
-    
-    public static let Unknown         = APIError(description: "Unknown Error")
-    public static let Unauthenticated = APIError(description: "Unauthenticated")
+    public static let Unknown         = APIError(description: "Unknown Error", code: nil)
+    public static let Unauthenticated = APIError(description: "Unauthenticated", code: nil)
```

Comment:
I don't get why this is needed. Isn't this something that is expected to be retrieved by the server? The other 2 cases may be used when there is no http response whatsoever. I would definitely have something for timeouts if there isn't something already there in the underlying network client.